torch/_inductor/lowering.py

```diff
@@ -2764,6 +2764,7 @@ def scatter_fallback(
     reduce_ty = "add" if fn == "aten.scatter_" else "sum"
     if (
         reduce not in {None, reduce_ty}
+        or (reduce == "sum" and self.get_device() == torch.device("cpu"))
```

Comment:
Can we add a configuration to opt in the codegen instead of fallback? With this, the current codegen for reduction with atomic-add can still be exercised.

Reply:
Yes, added.